qiskit/circuit/library/standard_gates/ms.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Global Mølmer–Sørensen gate."""
+
+from qiskit.util import deprecate_arguments
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.quantumregister import QuantumRegister
+
+
+class MSGate(Gate):
+    """Global Mølmer–Sørensen gate.
+
+    The Mølmer–Sørensen gate is native to ion-trap systems. The global MS can be
```

Comment:
see this https://iopscience.iop.org/article/10.1088/1367-2630/aaa398#njpaaa398eqn2 and equation 2

Reply:
i think \sum(XX) is correct